pkg/controllers/vdb/httpservercertgen_reconciler.go

```diff
@@ -54,10 +53,6 @@ func MakeHTTPServerCertGenReconciler(vdbrecon *VerticaDBReconciler, log logr.Log
 // Reconcile will create a TLS secret for the http server if one is missing
 func (h *HTTPServerCertGenReconciler) Reconcile(ctx context.Context, _ *ctrl.Request) (ctrl.Result, error) {
 	const PKKeySize = 2048
-	// Early out if the NMA isn't going to be used.
-	if !vmeta.UseVClusterOps(h.Vdb.Annotations) {
-		return ctrl.Result{}, nil
-	}
```

Comment:
As discussed in the scrum, we can't change the order for HTTPServerCertGenReconciler. We will leave it to generate HTTPServerCertGenReconciler for all AT database versions.